glTF-BinExporter/RhinoDocGltfConverter.cs

```diff
@@ -107,20 +130,13 @@ private void AddRhinoObjectDraco(Rhino.Geometry.Mesh[] rhinoMeshes, Rhino.DocObj
 
             var primitives = new List<MeshPrimitive>();
 
-            var transformYUp = Transform.Identity;
-            // Leave X as is
-            // Change Y to Z
-            transformYUp.M11 = 0;
-            transformYUp.M12 = 1;
-
-            // Change Z to -Y
-            transformYUp.M21 = -1;
-            transformYUp.M22 = 0;
-
             // For each rhino mesh, create gl-buffers, gl-meshes, etc.
             foreach (var rhinoMesh in rhinoMeshes)
             {
-                rhinoMesh.Transform(transformYUp);
+                if(options.MapRhinoZToGltfY)
+                {
+                    rhinoMesh.Transform(ZtoYUp);
+                }
                 rhinoMesh.TextureCoordinates.ReverseTextureCoordinates(1);
```

Comment:
Should the texture coordinate remapping happen, if we don't transform z to y up?

Reply:
I'm quite sure. Here's what happens if I don't.

![image](https://user-images.githubusercontent.com/16514346/107978147-fcc56480-6fc4-11eb-90bf-40900491042c.png)

When a mesh is transformed the texture coordinates themselves aren't changed. The texture coordinates are 2d points that correspond to locations on a texture. 

Here's the model: [Guitar.zip](https://github.com/Stykka/glTF-Bin/files/5983238/Guitar.zip)